src/traces/scattermapbox/attributes.js

```diff
@@ -87,6 +87,23 @@ module.exports = overrideAll({
                 'are only available for *circle* symbols.'
             ].join(' ')
         },
+        angle: {
```

Comment:
Just a note: I was pondering `angle` vs `rotation` as the attribute name, we have both in our schema... bar traces have a `textangle`, cartesian axes have a `tickangle`, but pie traces and angular axes have `rotation`. I don't see a clear pattern, nor a clear advantage of one over the other for this case, so let's keep `angle`.